src/lv_objx/lv_spinbox.h

```diff
@@ -47,6 +47,7 @@ typedef struct
     int32_t range_max;
     int32_t range_min;
     int32_t step;
+	bool rollover;	// Set to true for rollover functionality
```

Comment:
Please use `uint8_t rollover : 1` here. The API functions can still take/return `bool` as an argument.

Reply:
Updated!